### ISSUE TITLE
App is crashing on version 1.3.1 installed from F-droid on Android 11

### ISSUE THREAD

**Issue by AkechiShiro** [dkanada/gelli#164]
Hi,

I've just done the update today and the app crashes instantly at startup, it shows a loading circle then crashes in about a second.

My device is : Xiaomi Poco F3 (fairly recent) under Android 11, running on the stock ROM MIUI12 from the official manufacturer of the phone.

If need be I can provide a logcat and try to see if something shows up when launching Gelli.

Gelli worked perfectly fine on version 1.3.0.

**Comment by dkanada** [comment]
There is a newer update here on GitHub if you're willing to try that one out - it has several possible bug fixes. Logs would also be extremely helpful if you can gather them so I can narrow down the problem.

**Comment by brawnswubb** [comment]
Hello, not sure if this is allowed(I'm not sure how any of this works), but I had the same issue trying to play certain songs on 1.3.1 and 1.3.0(or whatever the previous version was). I came here to place an issue before stumbling on this and I followed @dkanada's advice with much success. All my 'trouble' songs are now playing with no crashes. Highly recommend giving it a try. Sorry if this is out of place
